### PR TITLE
Initial commit of lzv-rights project.

### PR DESCRIPTION
**Pull request by Enzewon**
Here is separate `pull request` that touches only `lzv-rights` project.

@artemilchuk – please, give final review before merging and deploying.

@chudaol – can you review this code? It doesn't differ from previous version, but you know, we can miss something. Lots of love ❤️

### CODE REVIEW

File: templates/includes/lzv-footer.jade [hunk 1]
```diff
@@ -0,0 +1,52 @@
+section
```
[the remark this answers is not in the document]
Comment (artemilchuk) [reply templates/includes/lzv-footer.jade]: I think that all included components must be in include folder.

File: css/others.scss [hunk 2]
```diff
@@ -1689,3 +1689,87 @@ p.section_news {
 		line-height: 22px;
 	}
 }
+
+.lzv-cover {
```
[the remark this answers is not in the document]
Comment (Enzewon) [reply css/others.scss]: Yes, we can divide it.

File: templates/includes/lzv-footer.jade (hunk 1 above)
[the remark this answers is not in the document]
Comment (Enzewon) [reply templates/includes/lzv-footer.jade]: Seems like it'll in future. @artemilchuk said that we're going to make unique footer for each page.

File: css/others.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (chudaol) [reply css/others.scss]: will you do it in this `PR`?

File: templates/includes/lzv-footer.jade (hunk 1 above)
[the remark this answers is not in the document]
Comment (chudaol) [reply templates/includes/lzv-footer.jade]: ping?

File: css/others.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (Enzewon) [reply css/others.scss]: Yes, I can do it in this `PR`.

**Comment by artemilchuk** [comment]
Guys! I finished work with lzv-project.

1) transport tlzv-footers to lzv-rights folder and add language navigation
2) transport lzv-project styles to lzv-rights.css

Inna tested lzv-project on https://22450592.ngrok.io/lzv-rights/
@chudaol — can you check and deploy this until Monday 10 a.m. Sincerely yours, Artem!

**Pull request merged**